### DISCUSSION TITLE
Huge console warn in unit tests

### DISCUSSION THREAD

**Discussion by aquibbaig** [pmndrs/zustand#1612] [General]
## Summary
I'm getting a huge list of `console.warn` for every created store in my repository when running my tests. The log looks like this:
```
 console.warn
    [zustand devtools middleware] Please install/enable Redux devtools extension

      24 | };
      25 |
    > 26 | export const myStore = create<myStore>(
         |                                        ^
      27 |   devtools((set) => {
```

zustand version: `"zustand": "^3.6.9"`

## Check List

Please do not ask questions in issues.

- [ ] I've already opened a [discussion](https://github.com/pmndrs/zustand/discussions) before opening this issue, or already discussed in other media.

Please include a minimal reproduction.

- [ ] I've added a link to a [typescript playground](https://www.typescriptlang.org/play) or [codesandbox](https://codesandbox.io) with a minimal reproduction.

Please check this if you're filing an issue regarding TypeScript.

- [ ] I've read the [typescript guide](https://github.com/pmndrs/zustand/blob/main/docs/guides/typescript.md), in particular that `create` is to be used as `create<T>()(...)` and not `create<T>(...)`.

**Comment by dai-shi** [comment]
Does it reproduce with the latest version of zustand?

It has `enabled` option, and you could set `false` in test env, which avoids showing the warning.
